cmd/meroxa/root/pipelines/create_test.go

```diff
@@ -66,7 +66,7 @@ func TestCreatePipelineFlags(t *testing.T) {
 		hidden    bool
 	}{
 		{name: "metadata", required: false, shorthand: "m", hidden: false},
-		{name: "env", required: false, hidden: true},
+		{name: "env", required: false, hidden: false},
```

Comment:
I'm guilty of this because of so many other attributes I added that are also `false`, but maybe we could simply start removing them altogether. What do you think?

Reply:
that's fair to not have to specify false when the default is false. updated here.